### ISSUE TITLE
Cannot find/tap table view delete "widget"

### ISSUE THREAD

**Issue by blackgold9** [kif-framework/KIF#58]
I'm attempting to tap the red circle with the line through it that is part of the delete table row style.  The accessibility inspector clearly shows it to be an element labeled  "Delete", but KIF is unable to find it. Am i missing something?
